gate_processing/make_gate_l2.py

```diff
@@ -81,66 +82,63 @@ def process_gate(fdir):
         )
     )
 
-    sondes = (
-        sondes.assign(
-            p=(
-                sondes.p.dims,
-                (sondes.p * 100).values,
-                {
-                    "standard_name": "air_pressure",
-                    "units": "Pa",
-                    "long_name": "atmospheric pressure",
-                },
-            ),
-            q=(
-                sondes.q.dims,
-                (sondes.q / 1000).values,
-                {
-                    "standard_name": "specific_humidity",
-                    "units": "kg/kg",
-                    "long_name": "specific humidity",
-                },
-            ),
-            ta=(
-                sondes.ta.dims,
-                (sondes.ta + 273.15).values,
-                {
-                    "standard_name": "air_temperature",
-                    "units": "K",
-                    "long_name": "air temperature",
-                },
-            ),
-        )
-        .assign(
-            rh=(
-                sondes.q.dims,
-                mt.specific_humidity_to_relative_humidity(
-                    sondes.q, sondes.p, sondes.ta, es=es
-                ).values,
-                {
-                    "standard_name": "relative_humidity",
-                    "units": "'",
-                    "long_name": "relative humidity with respect to liquid",
-                    "description": "Wagner-Pruss saturation vapor pressure",
-                },
-            ),
-            theta=(
-                sondes.ta.dims,
-                mt.theta(sondes.ta, sondes.p).values,
-                {
-                    "standard_name": "air_potential_temperature",
-                    "units": "K",
-                    "long_name": "dry potential temperature",
-                },
-            ),
-        )
-        .set_coords(["launch_lat", "launch_lon"])
+    sondes = sondes.assign(
+        p=(
+            sondes.p.dims,
+            (sondes.p * 100).values,
+            {
+                "standard_name": "air_pressure",
+                "units": "Pa",
+                "long_name": "atmospheric pressure",
+            },
+        ),
+        q=(
+            sondes.q.dims,
+            (sondes.q / 1000).values,
+            {
+                "standard_name": "specific_humidity",
+                "units": "kg/kg",
+                "long_name": "specific humidity",
+            },
+        ),
+        ta=(
+            sondes.ta.dims,
+            (sondes.ta + 273.15).values,
+            {
+                "standard_name": "air_temperature",
+                "units": "K",
+                "long_name": "air temperature",
+            },
+        ),
     )
+    sondes = sondes.assign(
+        rh=(
+            sondes.q.dims,
+            mt.specific_humidity_to_relative_humidity(
+                sondes.q, sondes.p, sondes.ta, es=es
+            ).values,
+            {
+                "standard_name": "relative_humidity",
+                "units": "'",
```

Comment:
The [CF Conventions](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.12/cf-conventions.html#dimensionless-units) suggest to use `1` as unit for dimensionless variables
```suggestion
                "units": "1",
```